launch/13-diffdrive.launch.py

```diff
@@ -0,0 +1,73 @@
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.actions import IncludeLaunchDescription
+from launch.substitutions import LaunchConfiguration
+from launch_ros.actions import Node
+from launch.actions import ExecuteProcess
+from launch_ros.substitutions import FindPackageShare
+
+def generate_launch_description():
+    package_arg = DeclareLaunchArgument('urdf_package',
+                                        description='The package where the robot description is located',
+                                        default_value='urdf_sim_tutorial')
+    model_arg = DeclareLaunchArgument('urdf_package_path',
+                                      description='The path to the robot description relative to the package root',
+                                      default_value='urdf/13-diffdrive.urdf.xacro')
+
+    rvizconfig_arg = DeclareLaunchArgument(
+        name="rvizconfig",
+        default_value=[FindPackageShare('urdf_sim_tutorial'), '/rviz/odom_urdf.rviz'],
```

Comment:
`os.path.join`

Reply:
Why use `os.path.join` when launch_ros will concatenate them together for you, and then you don't have to worry about sending a `launch.Substitution` to `os.path.join`

Reply:
I was refering to this:
```python
        default_value=[FindPackageShare('urdf_sim_tutorial'), os.path.join('rviz', 'odom_urdf.rviz')],
```